drawee/src/main/java/com/facebook/drawee/view/GenericDraweeView.java

```diff
@@ -282,6 +285,9 @@ private void inflateHierarchy(Context context, @Nullable AttributeSet attrs) {
                 R.styleable.GenericDraweeView_roundingBorderPadding,
                 roundingBorderPadding);
 
+          } else if (idx == R.styleable.GenericDraweeView_imageUri) {
```

Comment:
This attribute should be moved from `GenericDraweeView` attributes to `SimplaDraweeView` attributes (which don't exist at the moment so it would have to be added).